ivy/functional/ivy/gradients.py

```diff
@@ -400,6 +426,63 @@ def optimizer_update(
     -------
     ret
         The new function weights ws_new, following the optimizer updates.
+    
+    Functional Examples
+    -------------------
+    With :code:`ivy.Array` inputs:
+    
+    >>> w = ivy.array([1., 2., 3.])
+    >>> effective_grad = ivy.zeros(3)
+    >>> lr = 3e-4
+    >>> wsnew = ivy.optimizer_update(w=w, effective_grad=effective_grad, lr=lr,\
```

Comment:
use `ws_new`, we should have underscores between words in a variable name.

Reply:
Sure I will commit for this change